docs/release_notes.md

```diff
@@ -1,5 +1,39 @@
 # Release Notes
 
+## Version 1.1.0
+
+### Bug Fixes
+
+- Filtered characters to get valid names for nodes and attributes since some special characters are not allowed in Maya for Linux. *AdonisFX-1033*
+- Fixed Python runtime error due to incorrect element selection on node creation. *AdonisFX-1031*
```

Comment:
I would not say "runtime errors", for me seems that we are giving a bad a image of the product.

```suggestion
- Fixed element selection check at node creation. *AdonisFX-1031*
```

Reply:
I agree, done.